### ISSUE TITLE
Map panel doesn't show user picture with device tracker

### ISSUE THREAD

**Issue by hmmbob** [home-assistant/frontend#9005]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The Map panel doesn't use the user-picture of a person's associated device-tracker, but an initial.

![image](https://user-images.githubusercontent.com/33529490/116264235-6bb7fa00-a77a-11eb-8723-3d47644a0082.png)

### Describe the behavior you expected

Map panel using the user/person picture for device trackers, when this picture is available.

### Steps to reproduce the issue

1. Create a person
2. Add an user image to this person
3. Assign an existing device tracker to this person
4. Check the map
...

### What version of Home Assistant Core has the issue?

core-2021.4.6

### In which browser are you experiencing the issue with?

Chrome 90.0.4430.85

### Which operating system are you using to run this browser?

Win10 20H2

### State of relevant entities

```txt
## person.bob:

editable: false
id: 8b372c9d571b4133982db92cd4075bbc
latitude: 51.44180950128621
longitude: 5.467367582023143
gps_accuracy: 1
source: device_tracker.sm_a505fn
user_id: 510b1afef60640d7a220711c50b2192a
friendly_name: Bob
icon: mdi:human-male
entity_picture: /api/image/serve/5458e6d516076bc6166d39282d0c06fb/512x512

## device_tracker.sm_a505fn:

source_type: gps
latitude: 51.44180950128621
longitude: 5.467367582023143
gps_accuracy: 1
altitude: 65
course: 0
speed: 0
vertical_accuracy: 0
friendly_name: Bob
```

### Additional information

AFAIK there is no easy way to assign this user picture to the device_tracker

**Comment by arenasa70** [comment]
The image on the map is not the Person Image but the tracker image.  You can add an entity_picture value on Customize for that tracker.  Just select Other ,
attribute : entity_picture 
value: /api/image/serve/5458e6d516076bc6166d39282d0c06fb/512x512
This is working for me, but I'm using a SVG image (not supported for person) stored locally. 
I use: entity_picture: /local/images/person_male.svg
That works for me.

**Comment by hmmbob** [comment]
Yeah, that's not really an easy way to do so though, because it's not trivial to find the correct image. This should be more straight forward.

**Comment by hmmbob** [comment]
Ok, I was playing around with the dev-tools in my browser a bit an figured out that actually the `device_tracker` entity _and_ the `person` entity are plotted at the exact same location (which makes sense), but that the `device_tracker` one is always on top. If we could just 'prioritize' the `person` entity (so that it always shows on top of other entities) I guess the issue would be resolved 🤷🏻

**Comment by hmmbob** [comment]
Closed by #9331
